Defer the sequence data from being loaded by default

ttung opened the pull request:
### Description
Sequence data is too big to load by default.

Too bad sqlalchemy 1.4 is not yet release, as it has an awsesome [`defer(raiseload=True)`](https://docs.sqlalchemy.org/en/14/orm/loading_columns.html#raiseload-for-deferred-columns) feature.

### Test plan
gha.

phoenixAja commented on src/py/aspen/database/models/sequences.py:
```diff
@@ -146,7 +146,7 @@ class PathogenGenome(Entity):
 
     entity_id = Column(Integer, ForeignKey(Entity.id), primary_key=True)
-    sequence = Column(String, nullable=False)
+    sequence = deferred(Column(String, nullable=False))
```
so this is storing the actual sequence data?? i thought we would be storing an s3 key or something like that

ttung replied:
no, we're storing the sequence data in the db.

Commit:
Defer the sequence data from being loaded by default

### Description
Sequence data is too big to load by default.

Too bad sqlalchemy 1.4 is not yet release, as it has an awsesome [`defer(raiseload=True)`](https://docs.sqlalchemy.org/en/14/orm/loading_columns.html#raiseload-for-deferred-columns) feature.

### Test plan
gha.
